src/coreclr/jit/morph.cpp

```diff
@@ -859,6 +861,14 @@ fgArgTabEntry* fgArgInfo::AddRegArg(unsigned          argNum,
     curArgTabEntry->SetByteOffset(0);
 
     hasRegArgs = true;
+    if (argCount >= argTableSize)
```

Comment:
How frequently does this happen? Can we tell in advance (i.e., before first allocating this table) how many additional table entries we'll need? Do we know enough that we should just guess we might need more and over-allocate at the beginning?

Reply:
I guess you already have a "TODO" in lower.cpp about this.

Reply:
Yeah, it's not totally easy to make the exact determination in morph when we allocate the table, since we will only need an extra arg for indirect calls that go through the dispatcher, and even direct calls can end up being indirect depending on what the runtime returns to us.

We could probably handle some of the obvious cases, e.g. indirect calls without indirection cells on x64, but given that the feature is mainly planned to be turned on ahead-of-time I will opt to keep the source simple.